xchange-bittrex/src/main/java/org/knowm/xchange/bittrex/BittrexExchange.java

```diff
@@ -70,7 +75,15 @@ public void remoteInit() throws IOException {
       synchronized (INIT_LOCK) {
         if (bittrexSymbols.isEmpty()) {
           bittrexSymbols = ((BittrexMarketDataServiceRaw) marketDataService).getBittrexSymbols();
-          BittrexAdapters.adaptMetaData(bittrexSymbols, exchangeMetaData);
+          List<BittrexCurrency> bittrexCurrencies =
+                  ((BittrexMarketDataServiceRaw) marketDataService).getBittrexCurrencies();
+          Map<CurrencyPair, Fee> dynamicTradingFees = null;
+          try {
+            dynamicTradingFees = accountService.getDynamicTradingFees();
+          } catch (BittrexException | IOException e) {
+            // No connection or authentication ?
```

Comment:
silent errors are a bad practice

this exception should be let out or this method or in the least logged

Reply:
fixed